packages/client/src/settings/settings.ts

```diff
@@ -70,7 +70,7 @@ function findSettingsFiles(docUri?: Uri, isUpdatable?: boolean): Promise<Uri[]>
         .map((root) => configFileLocations.map((rel) => path.join(root, rel)))
         .reduce((a, b) => a.concat(b), []);
 
-    const found = possibleLocations.map(async (filename) => ({ filename, exists: await fs.pathExists(filename) }));
```

Comment:
Thank you for picking up on this.

Please use `fileExists` from 'common-utils/file.js';

Reply:
I looked into it, but `fileExists()` throws exceptions, which is the point if this PR.  Any reason why _not_ to use `fs.pathExists()`?

Reply:
`fs.existsSync` will block, slowing the search down.

Under normal circumstances, `fileExists` should not throw an exception. It catches the exception thrown by fs.access and returns `false`. 
https://github.com/streetsidesoftware/vscode-spell-checker/blob/main/packages/__utils/src/file.ts#L5